perennialAutodidact/memory-snap
# 38 formstep header component

## Description of changes

- Created and styled the header component for the setup form 

## Test steps
- Navigate to [http://localhost:3000/setup](http://localhost:3000/setup)
- Verify that the form header component loads and displays the "Game setup" header, "Step 1 of 4" and a progress bar that indicates the current step
